_opensearch/point-in-time-api.md

```diff
@@ -0,0 +1,270 @@
+            "creation_time": 1658146050064,
+            "keep_alive": 6000000
+        }
+    ]
+}
+```
+
+### Response fields
+
+Field | Data Type | Description 
+:--- | :--- | :---  
+pits | Array of JSON objects | The list of all PITs. 
+
+Each PIT object contains the following fields:
+
+Field | Data Type | Description 
+:--- | :--- | :---  
+pit_id | [Base64 encoded binary]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/binary) | The PIT ID.
+creation_time | long | The time the PIT was created in milliseconds since the epoch. 
+keep_alive | long |  The amount of time to keep the PIT in milliseconds.
+
+## Delete PITs
+Introduced 2.4
+{: .label .label-purple }
+
+Deletes one, several, or all PITs. PITs are automatically deleted when the `keep_alive` time period elapses. However, to free resources, you can delete a PIT using the Delete PIT API. The Delete PIT API supports deleting a list of PITs by ID or deleting all PITs at once.
+
+### Cross-cluster behavior
+
+The Delete PITs by ID API fully supports deleting cross-cluster PITs. 
+
+The Delete All PITs API deletes only local PITs or mixed PITs (PITs created in both local and remote clusters). It does not delete fully remote PITs. 
+
+#### Sample Request: Delete all PITs
+
+```json
+DELETE /_search/point_in_time/_all
+```
+
+If you want to delete one or several PITs, specify their PIT IDs in the request body.
+
+### Request fields
+
+Field | Data Type | Description  
+:--- | :--- | :---
+pit_id | [Base64 encoded binary]({{site.url}}{{site.baseurl}}/opensearch/supported-field-types/binary) or an array of binaries | The PIT IDs of the PITs to be deleted. Required.
+
+#### Sample request: Delete PITs by ID
+
+```json
+DELETE /_search/point_in_time
+
+{
+    "pit_id": [
+        "o463QQEPbXktaW5kZXgtMDAwMDAxFkhGN09fMVlPUkVPLXh6MUExZ1hpaEEAFjBGbmVEZHdGU1EtaFhhUFc4ZkR5cWcAAAAAAAAAAAEWaXBPNVJtZEhTZDZXTWFFR05waXdWZwEWSEY3T18xWU9SRU8teHoxQTFnWGloQQAA",
+        "o463QQEPbXktaW5kZXgtMDAwMDAxFkhGN09fMVlPUkVPLXh6MUExZ1hpaEEAFjBGbmVEZHdGU1EtaFhhUFc4ZkR5cWcAAAAAAAAAAAIWaXBPNVJtZEhTZDZXTWFFR05waXdWZwEWSEY3T18xWU9SRU8teHoxQTFnWGloQQAA"
+    ]
+}
+```
+
+#### Sample response
+
+For each PIT, the response contains a JSON object with a PIT ID and a `successful` field that specifies if the deletion was successful. Partial failures are treated as failures. 
+
+```json
+{
+    "pits": [
+        {
+            "successful": true,
+            "pit_id": "o463QQEPbXktaW5kZXgtMDAwMDAxFkhGN09fMVlPUkVPLXh6MUExZ1hpaEEAFjBGbmVEZHdGU1EtaFhhUFc4ZkR5cWcAAAAAAAAAAAEWaXBPNVJtZEhTZDZXTWFFR05waXdWZwEWSEY3T18xWU9SRU8teHoxQTFnWGloQQAA"
+        },
+        {
+            "successful": false,
+            "pit_id": "o463QQEPbXktaW5kZXgtMDAwMDAxFkhGN09fMVlPUkVPLXh6MUExZ1hpaEEAFjBGbmVEZHdGU1EtaFhhUFc4ZkR5cWcAAAAAAAAAAAIWaXBPNVJtZEhTZDZXTWFFR05waXdWZwEWSEY3T18xWU9SRU8teHoxQTFnWGloQQAA"
+        }
+    ]
+}
+```
+
+### Response fields
+
+Field | Data Type | Description | 
```

Comment:
```suggestion
Field | Data Type | Description
```

Reply:
Implemented, although this does not make a difference in rendering :)